HACKING.md

```diff
@@ -2,13 +2,35 @@
 
 To work in the framework itself you will need Python >= 3.8. Linting, testing,
 and docs automation is performed using
-[`tox`](https://tox.readthedocs.io/en/latest/), which you should install.
-For improved performance on the tests, ensure that you have PyYAML
-installed with the correct extensions:
+[`tox`](https://tox.readthedocs.io/en/latest/).
+
+First, make sure to install [uv](https://docs.astral.sh/uv/), for example via:
+
+```sh
+sudo snap install astral-uv --classic
+```
+
+Then install `tox` with extensions, as well as a range of Python versions:
```

Comment:
Is installing the range of Python versions necessary here?

Reply:
Let me try without.

My thinking was that we test for 3.8 / 10 / 12, but docs specifically need 11.

However, for quick start, the shortest sequence of commands that actually works is probably best.

Reply:
It's fine without. Thanks for the suggestion to simplify this :)